src/main/java/frc/robot/commands/DeepClimbPrep.java

```diff
@@ -46,14 +49,15 @@ public DeepClimbPrep(Climb climb, Arm arm, ScoringLimelight scoringLimelight, Ro
         new ConditionalCommand(
             new InstantCommand(),
             new SequentialCommandGroup(
-                new InstantCommand(()->arm.setDesiredPosition(ArmPosition.HOME)),
-                new WaitUntilCommand(() -> arm.atDesiredArmPosition())),
+                new InstantCommand(() -> arm.setDesiredPosition(ArmPosition.DEEP_CLIMB_PREP)),
+                new WaitUntilCommand(arm::atDesiredArmPosition),
+                new InstantCommand(() -> elevator.setDesiredPosition(ElevatorHeight.HOME)),
+                new WaitUntilCommand(elevator::atDesiredPosition)),
```

Comment:
move elevator first, wait for clear of climb, move arm, wait for both at desired